kolibri/plugins/coach/assets/src/state/actions/reports.js

```diff
@@ -34,11 +33,11 @@ const SummaryLogResource = coreApp.resources.ContentSummaryLogResource;
  * @returns {Promise} that resolves channel with lastActive value in object:
  *   { 'channelId': dateOfLastActivity }
 */
-function channelLastActivePromise(channel, classId) {
+function channelLastActivePromise(channel, userScope, userScopeId) {
   const summaryPayload = {
     channel_id: channel.id,
-    collection_kind: ReportConstants.UserScopes.CLASSROOM,
-    collection_id: classId,
+    collection_kind: userScope,
```

Comment:
That's an interesting point - I can see that causing confusion. It's actually named to contrast with "content scope" used elsewhere, which can be channel, topic, or content item.

Interested if there is a better pair of names?